src/Request/StreamExtensions.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Botwin.Request
+{
+    using System.IO;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public static class StreamExtensions
+    {
+        public static string AsString(this Stream stream)
+        {
+            using (var reader = new StreamReader(stream))
+            {
+                var readStream = reader.ReadToEnd();
+
+                if (stream.CanSeek)
+                {
+                    stream.Position = 0;
+                }
+
+                return readStream;
+            }
+        }
+
+        public static string AsString(this Stream stream, Encoding encoding)
```

Comment:
Going to be a pain and say make Encoding an optional argument with a default of UTF8 then we can have just the one method

Reply:
No pain.
I agree with you.
That it will make things even more simple.
Feel free to ask for any other improvement.